client/components/App.jsx

```diff
@@ -109,9 +112,12 @@ const ButtonRight = styled.div`
 ButtonRight.displayName = 'ButtonRight';
 
 const ButtonLeft = styled.div`
-  height: 35px;
-  width: 35px;
-  background-image: url('https://carousel-media.s3.us-east-2.amazonaws.com/carousel-media/Screen+Shot+2021-01-18+at+9.23.25+AM.png');
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  box-shadow: 0px 0px 2px #544d4d;
+  background-image: url('https://carousel-media.s3.us-east-2.amazonaws.com/carousel-media/pngegg.png');
+  background-size: 20px 20px;
   background-repeat: no-repeat;
```

Comment:
These two buttons looks good.
I was just curious, are the anyway you can dry this up and reuse some sort of general button and just pass the image url as a prop? Besides the image url and left being different, everything else looks the same to me. We can talk about it if you want.